tests/fixtures/scenarios/exitCode/components/terraform/hook-and-store/main.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  required_providers {
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.0"
+    }
+  }
+}
+
+variable "stage" {
+  description = "Stage. Used to define an Atmos stack."
+  type        = string
+  default     = "test"
+}
+
+resource "null_resource" "run_once" {
```

Comment:
```suggestion
resource "terraform_data" "force_error" {
  input = timestamp()

  lifecycle {
    precondition {
      condition     = false
      error_message = "Simulated Terraform error for testing."
    }
  }
}

```

Reply:
This always forced error. Found a better one now it is cross platform :)